src/System.IO.Ports/tests/SerialPort/ReadChar.cs

```diff
@@ -438,7 +438,8 @@ public void Read_Surrogate()
                     5000, "Err_91818aheid BytesToRead");
 
                 // We expect this to fail, because it can't read a surrogate
-                AssertExtensions.Throws<ArgumentException>(null, () => com1.ReadChar());
+                // parameter name can be different on different platforms
+                Assert.Throws<ArgumentException>(() => com1.ReadChar());
```

Comment:
Is there a parameter name to be checked here?

Reply:
I've removed that because it is a difference which is not coming from my code. Here is the full stack on Linux (paramName="chars"):

```
          /root/coreclr/src/System.Private.CoreLib/shared/System/Text/Encoding.cs(1299,0): at System.Text.Encoding.ThrowCharsOverflow()
          /root/coreclr/src/System.Private.CoreLib/shared/System/Text/Encoding.cs(1316,0): at System.Text.Encoding.ThrowCharsOverflow(DecoderNLS decoder, Boolean nothingDecoded)
          /root/coreclr/src/System.Private.CoreLib/shared/System/Text/UTF32Encoding.cs(971,0): at System.Text.UTF32Encoding.GetChars(Byte* bytes, Int32 byteCount, Char* chars, Int32 charCount, DecoderNLS baseDecoder)
          /root/coreclr/src/System.Private.CoreLib/shared/System/Text/DecoderNLS.cs(126,0): at System.Text.DecoderNLS.GetChars(Byte[] bytes, Int32 byteIndex, Int32 byteCount, Char[] chars, Int32 charIndex, Boolean flush)
          /root/coreclr/src/System.Private.CoreLib/shared/System/Text/DecoderNLS.cs(97,0): at System.Text.DecoderNLS.GetChars(Byte[] bytes, Int32 byteIndex, Int32 byteCount, Char[] chars, Int32 charIndex)
          /home/krwq/src/corefx/src/System.IO.Ports/src/System/IO/Ports/SerialPort.cs(719,0): at System.IO.Ports.SerialPort.ReadOneChar(Int32 timeout)
          /home/krwq/src/corefx/src/System.IO.Ports/src/System/IO/Ports/SerialPort.cs(637,0): at System.IO.Ports.SerialPort.ReadChar()
          /home/krwq/src/corefx/src/System.IO.Ports/tests/SerialPort/ReadChar.cs(441,0): at System.IO.Ports.Tests.ReadChar.<>c__DisplayClass17_0.<Read_Surrogate>b__1()
```